packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelMessageList.tsx

```diff
@@ -31,13 +33,31 @@ const GroupChannelMessageList = (props: GroupChannelProps['MessageList']) => {
   };
 
   // FIXME: Workaround, should run after data has been applied to UI.
-  const lazyScrollToIndex = (index = 0, animated = false, timeout = 0) => {
+  const lazyScrollToIndex = (index = 0, animated = false, timeout = 0, viewPosition = 0.5) => {
     setTimeout(() => {
-      ref.current?.scrollToIndex({ index, animated, viewPosition: 0.5 });
+      ref.current?.scrollToIndex({ index, animated, viewPosition });
     }, timeout);
   };
 
-  const scrollToMessage = useFreshCallback((createdAt: number, focusAnimated = false): boolean => {
+  const scrollToMessage = useFreshCallback<GroupChannelScrollToMessageFunc>((messageId, options) => {
```

Comment:
Seems it returns `boolean` but GroupChannelScrollToMessageFunc's return type is set to `void`?

Reply:
Oh thanks! you are right.